data/decks/web.yaml

```diff
@@ -1,6 +1,6 @@
 ---
-name: Web test deck without HTTP Invalid Request Line
-description: This deck runs HTTP Header Field Manipulation, and the Web Connectivity test
+name: Web test deck
+description: This deck runs HTTP Header Field Manipulation and the Web Connectivity test
```

Comment:
By the way, is `${probe_cc}` cached for a long time? I mean, how will `ooni-probe` behave if the laptop is suspended in one country and awaken in another?